retrofit/src/main/java/retrofit2/http/Field.java

```diff
@@ -30,21 +30,22 @@
  * field pair for each non-{@code null} item.
  * <p>
  * Simple Example:
- * <pre>
+ * <pre>{@code
```

Comment:
not sure if you want to bother, but Javadoc might format better with three spaces before `{@code` and a newline after it.

yes, this advice is ridiculous